pallets/admin-utils/src/lib.rs

```diff
@@ -764,6 +764,28 @@ pub mod pallet {
             T::Subtensor::set_weights_min_stake(min_stake);
             Ok(())
         }
+
+        /// Sets the minimum stake required for nominators , and clears small nominations
+        /// that are below the minimum required stake.
+        ///
+        /// # Arguments
+        ///
+        /// * `min_stake` - The minimum stake required for nominators.
+
+        #[pallet::call_index(43)]
+        #[pallet::weight((0, DispatchClass::Operational, Pays::No))]
+        pub fn sudo_set_nominator_min_required_stake(
+            origin: OriginFor<T>,
+            min_stake: u64,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+            log::info!("Setting minimum stake to: {}", min_stake);
+            T::Subtensor::set_nominator_min_required_stake(min_stake);
+            log::info!("Clearing small nominations");
+            T::Subtensor::clear_small_nominations();
```

Comment:
Only need to clear small nominations if `new_min_required_stake > prev_min_required_stake`.

```suggestion
            let prev_min_stake = T::Subtensor::get_nominator_min_required_stake();
            log::info!("Setting minimum stake to: {}", min_stake);
            T::Subtensor::set_nominator_min_required_stake(min_stake);
            if min_stake > prev_min_stake {
                log::info!("Clearing small nominations");
                T::Subtensor::clear_small_nominations();
            }
```